change/@fluentui-eslint-plugin-2d20c89f-0a2b-4e89-a47c-e6314bd42a2e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30293&commitId=817b9e8bb260c4707cc950c3490dffa28d57c9ad&env=prod&clientType=FLUENTUI)


  <details><summary><b>Chat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.Message Reactions With Popup Teams Dark.chromium.png|0| Removed|

</details>


  <details><summary><b>List</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.List Example Navigable Teams Dark.Focuses item.chromium.png|0| Removed|
|List.List Example Navigable Teams Dark.chromium.png|0| Removed|

</details>


  <details><summary><b>Pill</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pill.Pill Selectable Example.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>